doc/releases/migration-guide-3.6.rst

```diff
@@ -450,6 +450,18 @@ Bluetooth
     :kconfig:option:`CONFIG_BT_MESH_SAR_RX_DISCARD_TIMEOUT`,
     :kconfig:option:`CONFIG_BT_MESH_SAR_RX_ACK_RETRANS_COUNT` Kconfig options.
 
+* Audio
+
+  * The BT_AUDIO_CODEC_LC3_* values from ``<zephyr/bluetooth/audio/lc3.h>`` have moved to
```

Comment:
nit

```suggestion
  * The ``BT_AUDIO_CODEC_LC3_*`` values from ``<zephyr/bluetooth/audio/lc3.h>`` have moved to
```

Reply:
Wasn't sure about that one, since it's not an actual value, but I added your suggestion anyways :)